RxCocoa/iOS/UIScrollView+Rx.swift

```diff
@@ -57,6 +57,21 @@ extension Reactive where Base: UIScrollView {
         let source = RxScrollViewDelegateProxy.proxyForObject(base).contentOffsetPublishSubject
         return ControlEvent(events: source)
     }
+	
+	/// Reactive wrapper for delegate method `scrollViewDidEndDecelerating`
+	public var didEndDecelerating: ControlEvent<Void> {
+		let source = delegate.methodInvoked(#selector(UIScrollViewDelegate.scrollViewDidEndDecelerating(_:))).map { _ in }
+		return ControlEvent(events: source)
+	}
+	
+	/// Reactive wrapper for delegate method `scrollViewDidEndDragging(_:willDecelerate:)`
+	public var didEndDragging: ControlEvent<Bool> {
+		let source = delegate.methodInvoked(#selector(UIScrollViewDelegate.scrollViewDidEndDragging(_:willDecelerate:))).map { value -> Bool in
+			let willDecelerate = value[1] as! Bool
```

Comment:
We are also using 
```
return try castOrThrow(Bool.self, value[1])
```

for these conversions.

If we aren't using it this way somewhere, we should probably change that other place also.

This shouldn't ever fail, but if it does we probably shouldn't bring the entire app down because of this.